assets/sass/patterns/molecules/site-header-nav-bar-primary.scss

```diff
@@ -98,13 +108,32 @@
   }
 }
 
-@media only all and (max-width: 30rem) {
+@media only screen and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+  .nav-primary__alert_icon {
+    display: flex;
+    height: 20px;
+    width: 17px;
+  }
+
+  .nav-primary__item {
+    @include margin(24, "left");
+    display: flex;
```

Comment:
Does this need to be used here and not on nav-primary__item--list? Flex can be used once and should filter through the child elements.